discount/src/test/java/com/kyriba/conference/discount/DiscountDataMapperTest.java

```diff
@@ -0,0 +1,55 @@
+package com.kyriba.conference.discount;
+
+import com.kyriba.conference.discount.api.dto.DiscountExternalDto;
+import com.kyriba.conference.discount.api.dto.DiscountResponse;
+import com.kyriba.conference.discount.api.dto.DiscountUpdateParamsDto;
+import com.kyriba.conference.discount.domain.DiscountEntity;
+import com.kyriba.conference.discount.domain.DiscountType;
+import com.kyriba.conference.discount.util.DiscountDataMapper;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Igor Lizura
+ */
+public class DiscountDataMapperTest {
```

Comment:
Should be covered in other tests because mapping is used in business logic.

Reply:
Agree